Core/include/Acts/Seeding2/DoubletSeedFinder.hpp

```diff
@@ -76,18 +81,34 @@ class DoubletsForMiddleSp {
       return m_container->m_spacePoints[m_index];
     }
 
-    const LinCircle& linCircle() const {
-      return m_container->m_linCircles[m_index];
-    }
+    float cotTheta() const { return m_container->m_cotTheta[m_index]; }
+    float er() const { return m_container->m_er_iDeltaR[m_index][0]; }
+    float iDeltaR() const { return m_container->m_er_iDeltaR[m_index][1]; }
+    float u() const { return m_container->m_uv[m_index][0]; }
+    float v() const { return m_container->m_uv[m_index][1]; }
+    float x() const { return m_container->m_xy[m_index][0]; }
+    float y() const { return m_container->m_xy[m_index][1]; }
 
    private:
     const DoubletsForMiddleSp* m_container{};
     Index m_index{};
   };
+  class Proxy2 : public Proxy {
```

Comment:
I'm not exactly sure I follow, but this type essentially unpacks from the pair that's in the other proxy? If it's special-purpose, you could call it something more concrete.